conbench/tests/api/test_runs.py

```diff
@@ -323,12 +328,61 @@ def test_update_allowed_fields(self, client):
         response = client.put(f"/api/runs/{before.id}/", json=self.valid_payload)
         after = Run.one(id=before.id)
         self.assert_200_ok(response, _expected_entity(after))
+
         for key, value in self.valid_payload.items():
             if key == "finished_timestamp":
-                assert str(getattr(after, key)) == value
+                assert tznaive_dt_to_aware_iso8601_for_api(getattr(after, key)) == value
             else:
                 assert getattr(after, key) == value
 
+    @pytest.mark.parametrize(
+        "timestring",
```

Comment:
FYI: you can do something like `("input_timestring", "output_timestring")` as the first arg to `parametrize` and then include both of those in the function signature, and it will know how to unzip the pairs :)

Reply:
Thanks for reminding me! https://github.com/conbench/conbench/pull/626